com.unity.render-pipelines.universal/Documentation~/writing-shaders-urp-basic-unlit-structure.md

```diff
@@ -0,0 +1,177 @@
+            #pragma vertex vert
+            // This line defines the name of the fragment shader. 
+            #pragma fragment frag
+
+            // The Core.hlsl file contains definitions of frequently used HLSL
+            // macros and functions, and also contains #include references to other
+            // HLSL files (for example, Common.hlsl, SpaceTransforms.hlsl, etc.).
+            #include "Packages/com.unity.render-pipelines.universal/ShaderLibrary/Core.hlsl"            
+            
+            // The structure definition defines which variables it contains.
+            // This example uses the Attributes structure as an input structure in
+            // the vertex shader.
+            struct Attributes
+            {
+                // The positionOS variable contains the vertex positions in object
+                // space.
+                float4 positionOS   : POSITION;                 
+            };
+
+            struct Varyings
+            {
+                // The positions in this struct must have the SV_POSITION semantic.
+                float4 positionHCS  : SV_POSITION;
+            };            
+
+            // The vertex shader definition with properties defined in the Varyings 
+            // structure. The type of the vert function must match the type (struct)
+            // that it returns.
+            Varyings vert(Attributes IN)
+            {
+                // Declaring the output object (OUT) with the Varyings struct.
+                Varyings OUT;
+                // The TransformObjectToHClip function transforms vertex positions
+                // from object space to homogenous space
+                OUT.positionHCS = TransformObjectToHClip(IN.positionOS.xyz);
+                // Returning the output.
+                return OUT;
+            }
+
+            // The fragment shader definition.
+            half4 frag() : SV_Target
+            {
+                // Defining the color variable and returning it.
+                half4 customColor;
+                customColor = half4(0.5, 0, 0, 1);
+                return customColor;
+            }
+            ENDHLSL
+        }
+    }
+}
+```
+
+The shader paints the GameObject dark red (RGB value (0.5, 0, 0)).
+
+![The shader paints the GameObject dark red](Images/shader-examples/unlit-shader-tutorial-basic-hardcoded-color.jpg)
+
+The following section introduces you to the structure of this basic shader.
+
+<a name="basic-shaderlab-structure"></a>
+
+## Basic ShaderLab structure
+
+Unity shader assets are written in a Unity-specific language called [ShaderLab](https://docs.unity3d.com/Manual/SL-Shader.html). 
+
+The shader in this example has the following blocks:
+
+* [Shader](#shader)
+* [Properties](#properties)
+* [SubShader](#subshader)
+* [Pass](#pass)
+* [HLSLPROGRAM](#hlsl)
+
+<a name="shader"></a>
+
+### Shader block
+
+A ShaderLab file starts with the `Shader` declaration.
```

Comment:
Use either "A shader asset starts..." or "ShaderLab code starts..." - might be a bit nit-picky, but I want to limit the ways that we describe these things as it got very confusing in the past!